cupy/core/core.pyx

```diff
@@ -3526,7 +3541,20 @@ cpdef ndarray tensordot_core(
     # compute C^T = B^T * A here.
     a, transa, lda = _mat_to_cublas_contiguous(a, 0)
     b, transb, ldb = _mat_to_cublas_contiguous(b, 1)
-    if use_sgemmEx:
+    if use_tensor_core:
+        one_ptr = ctypes.cast(ctypes.byref(_one_fp32), ctypes.c_void_p)
```

Comment:
Please use `cdef` instead of `ctypes`, because `ctypes` is too slow.

Reply:
OK, will use cdef instead of ctypes there.